src/asset/asset.js

```diff
@@ -71,7 +71,7 @@ const VARIANT_DEFAULT_PRIORITY = ['pvr', 'dxt', 'etc2', 'etc1', 'basis'];
  * @property {pc.AssetRegistry} registry The asset registry that this Asset belongs to
  */
 class Asset extends EventHandler {
-    constructor(name, type, file, data = {}, options = {}) {
```

Comment:
Why default properties not preferred here?

Reply:
It's breaks texture loading and I have no idea why yet, so I backed it out for now to get this PR merged.